packages/block-editor/src/hooks/use-bindings-attributes.js

```diff
@@ -292,6 +280,8 @@ function shimAttributeSource( settings, name ) {
 	return {
 		...settings,
 		edit: withBlockBindingSupport( settings.edit ),
+		// TODO - move this to be located with pattern overrides code.
+		usesContext: [ 'patternOverridesContent', ...settings?.usesContext ],
```

Comment:
Should we move this to the `usesContext` property when registering the pattern overrides source? https://github.com/WordPress/gutenberg/blob/320dcb9633b2c2fcb5dd6ae99f517ce04e265e9e/lib/compat/wordpress-6.6/block-bindings/pattern-overrides.php#L57

Reply:
Will this also register `usesContext` for client side code?

Considering both `pattern/overrides` and `patternOverridesContent` are both sharing the same attribute via context, I guess they should be named the same thing. I'm not enamoured with using `pattern/overrides` naming  in js code where the variable is often accessed via dot notation or destructuring.

I think changing it would require a backport.

Reply:
> Will this also register usesContext for client side code?

If not, I think it could still be moved to this file if we change it over to the `registerBlockType` hook similar to the one in the `use-binding-attributes.js` file:
https://github.com/WordPress/gutenberg/blob/trunk/packages/editor/src/hooks/pattern-overrides.js#L110

Reply:
> Will this also register usesContext for client side code?

Yes, it should work client-side. We might make some changes to how it is handled, but it should work.

I'm not familiar with how that context is used, but it'd be great to use just one property if possible.

> I think changing it would require a backport.

Yes, I think it would need one. But it should be just one line.

Reply:
I decided to change it to `pattern/overrides` on the client as well, so that it's consistent and we won't need a backport.

I think I've addressed all the feedback.